stock_lot_quantity/README.rst

```diff
@@ -0,0 +1,31 @@
+Stock quantity for serial number/lot
+====================================
+
+This module proposes to compute the quantity available for each serial number (sum of quantities from the quants).
+The computation is limited to internals locations.
+
+Usage
+=====
+
+This module adds a field named `Quantity available` on the serial number tree and form.
+
+Credits
```

Comment:
Hi @lbellier thanks for your contribution
I will check mostly format and guidelines in your PR
Following https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst, I miss the bug tracker section and the Try me on Runbot button.